forest/src/daemon.rs

```diff
@@ -175,3 +218,31 @@ pub(super) async fn start(config: Config) {
 
     info!("Forest finish shutdown");
 }
+
+#[cfg(test)]
+mod test {
+    use super::*;
+    use db::MemoryDB;
+    use std::fs::File;
+    use std::io::BufReader;
+    use std::io::Read;
+
+    #[async_std::test]
+    async fn import_snapshot_from_file() {
+        let db = Arc::new(MemoryDB::default());
+        let file = File::open("chain4.car").expect("Snapshot file path not found!");
```

Comment:
I would move the binary file and test into own dir perhaps?

Reply:
Also, could use `include_bytes!()` instead to avoid io during test if you wanted (doesn't really matter tho since it's a tradeoff)

I would say for the car file at least move it from the forest base directory into a sub folder at least